pkg/policy/l4_filter_test.go

```diff
@@ -47,14 +47,12 @@ var (
 	bazSelector = api.NewESFromLabels(labels.ParseSelectLabel("baz"))
 
 	cachedFooSelector, _ = testSelectorCache.AddIdentitySelector(dummySelectorCacheUser, fooSelector)
-	cachedBarSelector, _ = testSelectorCache.AddIdentitySelector(dummySelectorCacheUser, barSelector)
```

Comment:
How come the  test doesn't need this line?

Reply:
cc @ianvernon

Reply:
Looks like it was added but never used by @jrajahalme . I don't see this as a big concern, it was probably left over after the tests were reworked after initially being written.